tasks/prelim.yml

```diff
@@ -92,6 +92,20 @@
       - audit
       - RHEL-07-010240
 
+- name: "PRELIM | Check /etc/sudoers and /etc/sudoers.d/* entries for NOPASSWD."
```

Comment:
I wouldn't put these in prelim.yml, I'd put them as part of the task within a named "block:", since these aren't needed in multiple tasks, though I know it's not consistent throughout the codebase.